lib/request/rack.rb

```diff
@@ -136,6 +136,7 @@ def if_modified_since
     accessor(:host,         Key.new('SERVER_NAME'))
     accessor(:query_string, Key.new('QUERY_STRING'))
     accessor(:content_type, Key.new('CONTENT_TYPE'), nil)
+    accessor(:cookies,      Key.new('HTTP_COOKIE'), EMPTY_STRING)
```

Comment:
@mbj fair enough, then what about depending on `snusnu/cookie` and having `#cookies` do `Cookie::Registry.coerce(header)` as a first step? (this is what this branch initially was doing btw)